frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx

```diff
@@ -52,7 +52,49 @@ import {
   getPort,
 } from '../kms-config/utils';
 import './ocs-storage-class-form.scss';
-import { GUARDED_FEATURES } from '../../features';
+
+export const CephFsNameComponent: React.FC<ProvisionerProps> = ({ onParamChange }) => {
+  const { t } = useTranslation();
+
+  const isExternal = useFlag(OCS_INDEPENDENT_FLAG);
+  const [fsName, setFsName] = React.useState<string>(
+    !isExternal ? OCS_STORAGECLUSTER_CEPHFILESYSTEM : '',
+  );
```

Comment:
```suggestion
const scName = `${isExternal ? OCS_EXTERNAL_CR_NAME : OCS_INTERNAL_CR_NAME}-cephfs`;
const [sc] = useK8sGet<StorageClassKind>(StorageClassModel, scName);
const fileSystemName = sc?.parameters?.fsName;
```
Use the `fileSystemName` as the default value. 
Feel free to add a loading state.